packages/mobile/src/features/videoPlayer/page/VideoPlayerScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { View} from "react-native";
+import Orientation from "react-native-orientation";
+import {NavigationProp, RouteProp } from "@react-navigation/native";
+import { YouTubePlayer } from "../commponents/YouTubePlayer";
+import { PlayerType } from "../models/PlayerType";
+
+
+
+type VideoPlayerRouteParam = {
+    playerType: PlayerType
+    videoUrl: string,
+    youtubeId:string
+}
+
+type Props = {
+  navigation: NavigationProp<any>;
+  route:RouteProp<any,any>
+};
+
+export class VideoPlayerScreen extends React.Component<Props,any> {
+
+  unsubscribeFocuse = ()=>{};
```

Comment:
unsubscribeFocus i pytanie czy to ma być puste

Reply:
Tak bo w trakcie działania kodu przypisuje tam lisner który potem próbuje zwolnić